### ISSUE TITLE
Define global variables in yaml file

### ISSUE THREAD

**Issue by einarsi** [equinor/septic-config-generator#163]
It should be possible to define global variables (available to all templates) in the yaml file. Global variables can be used to e.g. provide lists of wells for iteration inside templates. Such lists are currently often created when needed inside templates, and are usually defined multiple times. This makes it demanding to maintain.

This issue is somewhat related to #10

**Comment by palkitt** [comment]
One way of defining and using global variables with today's version is importing a "global" file in all files you need those global variables. Some snippets below:

Definition file (for global variables) "00_globals.cnfg":

`{% set wells = ['01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20', '23', '24', '28', '32', '35', '36']  %}`

Application file
```
{% import "00_globals.cnfg" as globals %}
...
{% for well in globals.wells %}
   "D{{ well }}" 
{%- endfor %}

```

**Comment by einarsi** [comment]
Yes, this is a nice technique that I also use myself. I still think it would be a nice feature to define such variables directly in the yaml config file.

**Comment by einarsi** [comment]
This may not be needed anymore after the introduction of globally available sources (#10) in v2.8.0. Closing for now until someone comes up with a use case where this is needed.
